build-system/tasks/lint.js

```diff
@@ -46,7 +46,7 @@ var buildSystemOptions = {
  * @return {!Array<string>}
  */
 function getBuildSystemFiles() {
-  if (!!process.env.TRAVIS_PULL_REQUEST_SHA) {
+  if (process.env.TRAVIS) {
```

Comment:
Don't we only want to diff the files for PR builds?

Reply:
You're right, we do. That logic is now in `pr-check.js`. Once we (eventually) fix all lint errors, we can get rid of some of this code and enforce lint checks across the entire codebase.